.vscode/settings.json

```diff
@@ -0,0 +1,20 @@
+{
+    "files.exclude": {
+        ".vscode/": true,
+        "node_modules/": true,
+        "src/tests/": true,
+        "LICENSE": true,
+        ".gitignore": true,
+        "package.json": true,
+        "package-lock.json": true,
+    },
+    "search.exclude": {
+        ".vscode/": true,
+        "node_modules/": true,
+        "src/tests/": true,
+        "LICENSE": true,
+        ".gitignore": true,
+        "package.json": true,
+        "package-lock.json": true,
+    }
+}
```

Comment:
This will hide all the listed files from the project explorer and search on vs code.
I'm using this to reduce noise (we're only showing the code and documentation files) and to prevent spoilers (by hiding the tests related files).